browser/help/topics/v4-hts.md

```diff
@@ -66,11 +66,549 @@ The above example will retrieve the entire frequency struct for each variant. To
+      - `bin_freq`: Bin frequencies for the histogram of AB in heterozygous individuals calculated on all genotypes. The number of records found in each bin.
+      - `n_smaller`: Count of AB values in heterozygous individuals falling below lowest histogram bin edge, calculated on all genotypes.
+      - `n_larger`: Count of AB values in heterozygous individuals falling above highest histogram bin edge, calculated on all genotypes.
+    - `age_hists`: Histograms containing age information for release samples.
+    - `age_hist_het`: Histogram for age in all heterozygous release samples calculated on high quality genotypes.
+      - `bin_edges`: Bin edges for the age histogram.
+      - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+      - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+      - `n_larger`: Count of age values falling above highest histogram bin edge.
+    - `age_hist_hom`: Histogram for age in all homozygous release samples calculated on high quality genotypes. If variant is in the pseudoautosomal regions of chrX or chrY, this histogram also includes age counts of hemizygous samples.
+      - `bin_edges`: Bin edges for the age histogram.
+      - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+      - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+      - `n_larger`: Count of age values falling above highest histogram bin edge.
+- `exomes`: Struct of allele frequency information from the gnomAD exomes.
+  - `freq`: Array of allele frequency information (AC, AN, AF, homozygote count) for each frequency aggregation group in the gnomAD exomes.
+    - `AC`: Alternate allele count in release.
+    - `AF`: Alternate allele frequency, (AC/AN), in release.
+    - `AN`: Total number of alleles in release.
+    - `homozygote_count`: Count of homozygous alternate individuals in release.
+  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency in the gnomAD exomes. Excludes Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining individuals" (`remaining`) groups.
+    - `AC`: Alternate allele count in the group with the maximum allele frequency.
+    - `AF`: Maximum alternate allele frequency, (AC/AN), across groups in gnomAD.
+    - `AN`: Total number of alleles in the group with the maximum allele frequency.
+    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
+    - `gen_anc`: Genetic ancestry group with maximum allele frequency
+  - `faf`: Filtering allele frequency in the gnomAD exomes.
+    - `faf95`: Filtering allele frequency (using Poisson 95% CI).
+    - `faf99`: Filtering allele frequency (using Poisson 99% CI).
```

Comment:
Better to keep as it is ordered in the joint HT: 
```suggestion
  - `faf`: Filtering allele frequency in the gnomAD genomes.
    - `faf95`: Filtering allele frequency (using Poisson 95% CI).
    - `faf99`: Filtering allele frequency (using Poisson 99% CI).
  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency in the gnomAD genomes. Excludes Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining individuals" (`remaining`) groups.
    - `AC`: Alternate allele count in the group with the maximum allele frequency.
    - `AF`: Maximum alternate allele frequency, (AC/AN), across groups in gnomAD.
    - `AN`: Total number of alleles in the group with the maximum allele frequency.
    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
    - `gen_anc`: Genetic ancestry group with maximum allele frequency.
  - `fafmax`: Information about the genetic ancestry group with the maximum filtering allele frequency.
    - `faf95_max`: Maximum filtering allele frequency (using Poisson 95% CI).
    - `faf95_max_gen_anc`: Genetic ancestry group with the maximum filtering allele frequency (95% CI).
    - `faf99_max`: Maximum filtering allele frequency (using Poisson 99% CI).
    - `faf99_max_gen_anc`: Genetic ancestry group with the maximum filtering allele frequency (99% CI).
```

Reply:
I started adding changes locally because the mix of committing here and local suggestions was making the formatting wonky -- let me know if you see anything weird slip through